unicorn/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+INCLUDE_DIRECTORIES(
+    ${PROJECT_SOURCE_DIR}/unicorn/include)
+
+ADD_LIBRARY(unicorn MODULE
+    include/cocaine/unicorn.hpp
```

Comment:
In fact it just helps CLion (IDE which I use) to properly handle headers. I'll remove them in PR